noxfile.py

```diff
@@ -6,6 +6,18 @@
 import shutil
 import nox
 
+@nox.session(py="3")
+def translation(session):
+    session.install("sphinx-intl")
+    session.install("sphinx-inline-tabs")
```

Comment:
This should probably reuse the same dep versions as the rest of the commands.
```suggestion
    session.install("-r", "requirements.txt")
```

Also, maybe we'd just add `sphinx-intl` into `requirements.txt` to avoid having it in just one place? Is it unnecessary in all the other envs?

Reply:
I think `sphinx-intl` is unnecessary now since Weblate will generate `.po` files for us when we add new languages to it.

Reply:
What about generating `messages.pot`? We still need to extract the original messages from time to time, right?

Reply:
yes, when people change the docs, they will have to run `nox -s translation` locally to update `messages.pot`.